lua/framework/lua_app.cc

```diff
@@ -59,6 +57,13 @@ LuaApp::Run(int argc, char** argv)
   opensn::Initialize();
   console.PostMPIInfo(opensn::mpi_comm.rank(), opensn::mpi_comm.size());
 
+  opensn::log.Log() << opensn::name << " version " << GetVersionStr();
+  opensn::log.Log() << Timer::GetLocalDateTimeString() << " Running " << opensn::name
+                    << " in interactive-mode with " << opensn::mpi_comm.size() << " processes.";
```

Comment:
Originally, OpenSn was printing if it runs in interactive or batch mode. This changed here and reports only interactive (even for the batch mode). However, is that really needed? May be we just print:
```
<time> Running OpenSn with <n> processors.
```

If you wanted to keep batch/interactive, you need to look at `sim_option_interactive_` and print either `interactive` or `batch` here.

Reply:
Nice catch. I removed the interactive piece as you suggested.